### DISCUSSION TITLE
Future Energy Prices does not handle negative prices (very well)

### DISCUSSION THREAD

**Discussion by ClausSamuelsen** [UtilitechAS/amsreader-firmware#570] [General]
<img width="1558" alt="Screenshot 2023-05-16 at 19 08 16" src="https://github.com/UtilitechAS/amsreader-firmware/assets/96246629/7595fc9d-3bc8-41a7-83ec-e8ef5cc28f3e">
As you can see on this screenshot there's hours where the electricity price is negative. But the graph is not capable of showing this.
Would be nice to have this capability in a future release
Best regards
Claus

**Comment by ArnieO** [comment]
Yes, and you're not the first to notice! 😉 See https://github.com/UtilitechAS/amsreader-firmware/issues/560

**Comment by gskjold** [comment]
Will be fixed in an upcoming patch 
![Screenshot from 2023-05-20 21-56-49-1](https://github.com/UtilitechAS/amsreader-firmware/assets/4446828/b8825a11-188f-4dd5-8b63-627c217c9af4)
